security/dirtycow.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+container_name="dirtycow"
+runtime=${RUNTIME:-kata-runtime}
```

Comment:
Do we need this? We set the default RUNTIME in a number of places, but tbh, I'm not sure how or if we set it 'globally'. I think having less of these default RUNTIME sets is probably the way forwards though. /cc @chavafg

Reply:
At least for the CI, this is not needed as it is exported on the `.ci/run.sh`. But it may be needed if you are only running this test in a local environment. So I think it is ok.

Reply:
I prefer to keep it, to make life easier to users that just want to check if their custom kernel is secure